metaseq/launcher/slurm.py

```diff
@@ -461,11 +539,16 @@ def launch_train(args, grid, grid_product, dry_run, postprocess_hyperparams):
                 )
             else:
                 write_git_commit(train_log)
+                if args.rsc:
+                    internal.klist(train_log)
                 with open(train_log, "a") as train_log_h:
                     job_id, stdout = run_batch(env, sbatch_cmd_str, sbatch_cmd)
                     print(stdout, file=train_log_h)
         if job_id is not None:
             print("Launched {}".format(job_id))
+        if hasattr(args, "tombstonable"):
+            if args.tombstonable:
+                tombstones(job_id=job_id, base_dir=args.default_prefix)
```

Comment:
is default_prefix the right thing here? Should it be the actual prefix of the job?

I'm still not sure we're checking for tombstone logic in the right place. I'm not sure this will work on RSC as is

Reply:
The name is just for consistency, but it is the actual prefix of the job. It results in the right path that Susan wrote here: https://github.com/fairinternal/metaseq-internal/pull/564#issuecomment-1300799020

Reply:
Changed naming of the arg to make it clearer what it is.